resources/views/community/components/global-statistics/recent-game-progress.blade.php

```diff
@@ -0,0 +1,68 @@
+@props([
+    'game',
+    'headingLabel' => 'Most recent game mastered',
+    'timestamp' => '',
+    'userId' => 1,
+])
+
+<?php
+use App\Site\Models\User;
+
+$user = User::find($userId);
+
+if (!$user) {
+    return null;
+}
+
+$system = $game->console->Name;
+
+$renderedGameTitle = renderGameTitle($game->Title);
+$gameUrl = route('game.show', $game->ID);
+?>
+
+<div>
+    <div class="text-2xs flex lg:justify-between items-center gap-x-2 w-full mb-0.5 flex-nowrap">
+        <p>{{ $headingLabel }}</p>
+        <p>({{ $timestamp }})</p>
+    </div>
+
+    <div class="text-xs bg-embed p-4 rounded border border-embed-highlight">
+        <div class="flex gap-x-2.5 leading-4 relative">
+            <!-- Keep the image and game title in a single tooltipped container. -->
+            <a
+                href="{{ $gameUrl }}"
+                x-data="tooltipComponent($el, {dynamicType: 'game', dynamicId: '{{ $game->ID }}'})"
+                @mouseover="showTooltip($event)"
+                @mouseleave="hideTooltip"
+                @mousemove="trackMouseMovement($event)"
+            >
+                <img
+                    src="{{ media_asset($game->ImageIcon) }}"
+                    alt="{{ $game->Title }} badge"
+                    width="64"
+                    height="64"
+                    class="w-16 h-16 min-w-[64px] rounded-sm"
+                >
+
+                <p class="absolute pl-4 top-[-2px] left-[58px] max-w-fit line-clamp-2 mb-px">
+                    {!! $renderedGameTitle !!}
+                </p>
+            </a>
+
+            <div class="-mt-1 w-full">
+                <!-- Provide invisible space to slide the console underneath -->
+                <p class="invisible max-w-fit line-clamp-2 mb-px">
+                    {!! $renderedGameTitle !!}
```

Comment:
Should the console line have an icon?
![image](https://github.com/RetroAchievements/RAWeb/assets/32680403/023ab5a5-c79e-4ced-a19b-8bf827e8e1be)

Reply:
I lean towards no, but happy to hear if you have a different opinion. My reasoning is for games that don't have 2 lines of content in their name, we have just enough space for the content to be flush against the game icon. If we add a console icon, the line height of the console name must increase. To maintain the "flush" effect, we'd need to compensate by increasing the game image size, which I don't think we should do.

Reply:
I was just aiming for consistency, but I didn't notice the sets widget doesn't have icons either. This is fine.